infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -937,22 +965,27 @@ public void append(long address, LogData entry) {
         } catch (IOException e) {
             log.error("Disk_write[{}]: Exception", address, e);
             throw new RuntimeException(e);
+        } finally {
```

Comment:
I tested this with the l2 tests and lsof doesn't show any deleted entries. Let's see what happens on the other system